assets/queries/terraform/aws_bom/rds/query.rego

```diff
@@ -0,0 +1,114 @@
+package Cx
+
+import data.generic.common as common_lib
+import data.generic.terraform as tf_lib
+
+CxPolicy[result] {
+	resource := input.document[i].resource.aws_db_instance[name]
+
+	accessibility := get_accessibility(resource)
+
+	bom_output = {
+		"resource_type": "aws_db_instance",
+		"resource_name": tf_lib.get_specific_resource_name(resource, "aws_db_instance", name),
+		"resource_accessibility": accessibility,
+		"resource_encryption": get_db_instance_encryption(resource),
+		"resource_vendor": "AWS",
+		"resource_category": "Storage",
+	}
+
+	final_bom_output = common_lib.get_bom_output(bom_output, "")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("aws_db_instance[%s]", [name]),
+		"issueType": "BillOfMaterials",
+		"keyExpectedValue": "",
+		"keyActualValue": "",
+		"searchLine": common_lib.build_search_line(["resource", "aws_db_instance", name], []),
+		"value": json.marshal(final_bom_output),
+	}
+}
+
+CxPolicy[result] {
```

Comment:
I think this policy is incomplete or irrelevant. Furthermore, we should have the same approach in TF and CF